### DISCUSSION TITLE
restore basic configuration

### DISCUSSION THREAD

**Discussion by felipe300** [LazyVim/LazyVim#400] [General]
I think i mess up my configuration, but i' not sure where, so i want to restore the basic configuration of LazyVim, there is a way to do that or i just have to uninstall Lazy and start again??? Thanks

**Comment by folke** [comment]
All your customizations are under `~/.config/nvim/lua`, so comment/updated any of the changes in those files.

**Reply by felipe300** [reply]
Thanks!!
